### PR TITLE
chore: ZIP/postcode/postal code messaging consistency

### PR DESCRIPTION
**Pull request by frosso**
## Description

Bringing more consistency on the "ZIP code"/"Postal Code" messaging

## Fixes

Fixes https://github.com/Automattic/woocommerce-services/issues/2091

## Steps to reproduce

Carrier connection:
Screenshot on 2020-08-06 at 16-04-45

Address selection (destination/source):
Annotation on 2020-08-06 at 16-31-52

### CODE REVIEW

File: classes/class-wc-connect-taxjar-integration.php [hunk 1]
```diff
@@ -315,15 +315,15 @@ public function _error( $message ) {
 		$invalid_postcode = false !== strpos( $formatted_message, 'isn\'t a valid postal code for' );
 		if ( ! is_admin() && ( $state_zip_mismatch || $invalid_postcode ) ) {
 			$fields = WC()->countries->get_address_fields();
-			$postcode_field_name = __( 'Postcode / ZIP', 'woocommerce-services' );
```
Comment (frosso) [inline classes/class-wc-connect-taxjar-integration.php:318]: This I wasn't sure if I had to modify as well 😬  Should I revert the changes on this file?

Comment (bborman22) [reply classes/class-wc-connect-taxjar-integration.php:318]: I think this makes sense to update as well. Only thing I noticed was this change has spaces around the slash while the other instances does not have spaces.

**Comment by bborman22** [comment]
This looked good to me and I confirmed all the labels and messages were updated as expected to ZIP/Postal code. One thing I wasn't sure of is the translation files for WCS. I noticed there was a [generated translation files](https://github.com/Automattic/woocommerce-services/tree/develop/i18n) that only has the Postal code msgid, so that may need to be re-generated to pick up the ZIP/Postal code msgid. Maybe @harriswong or @Ferdev have more information on the translation set up for WCS and could weigh in on any translation impact of this update?

File: classes/class-wc-connect-taxjar-integration.php (hunk 1 above)
Comment (frosso) [reply classes/class-wc-connect-taxjar-integration.php:318]: Yeah, the existing change had spaces around the slash - do you think this should be converted to _not_ have spaces?

Comment (bborman22) [reply classes/class-wc-connect-taxjar-integration.php:318]: Yeah I think for consistency it shouldn't have spaces. Given the overall goal of the update, I'd put more weight on matching the new messaging over matching any previous messaging.

File: classes/class-wc-connect-taxjar-integration.php [hunk 2]
```diff
@@ -315,15 +315,15 @@ public function _error( $message ) {
...
+			$postcode_field_name = __( 'ZIP / Postal code', 'woocommerce-services' );
```
Comment (frosso) [inline classes/class-wc-connect-taxjar-integration.php:318]:
```suggestion
			$postcode_field_name = __( 'ZIP/Postal code', 'woocommerce-services' );
```

File: classes/class-wc-connect-taxjar-integration.php (hunk 1 above)
Comment (frosso) [reply classes/class-wc-connect-taxjar-integration.php:318]: Sure thing, that makes sense 👍  Done in 401a709

**Comment by harriswong** [comment]
> I noticed there was a generated translation files that only has the Postal code msgid, so that may need to be re-generated to pick up the ZIP/Postal code msgid. Maybe @harriswong or @Ferdev have more information on the translation set up for WCS and could weigh in on any translation impact of this update?

Good question. We usually update our translation files per release through a script. Recently, we added a new script to generate these files: https://github.com/Automattic/woocommerce-services/blob/develop/package.json#L13.

**Comment by harriswong** [comment]
This process may change, but as far as I know, we don't need to regenerate language file per PR. Using the `__()` and `translate()` is good.
